### ISSUE TITLE
Rotation of chairs and desks

### ISSUE THREAD

**Issue by ImMX96** [MrCrayfish/MrCrayfishFurnitureMod#670]
I am helping create a small server to use as a mock government, and we chose this mod for the tables and chairs. However, I was wondering if it was possible to get the desks and chairs on an angle. So instead of being in line with the block, it would be angled. I looked this up to find that rotation is possible, but I am not sure if this is an issue or if it was never intended to rotate onto an angle,

TLDR: Is it a bug that I cannot have objects on an angle? or is that how the mod was made.

**Comment by MrCrayfish** [comment]
It's not a bug. It's just how I've decided how the furniture work. Much like other blocks in Minecraft, they can only be placed parrallel with an axis and not inbetween. It's suppose to be simple.

**Comment by MrCrayfish** [comment]
Undertstandably there are mods/datapacks that add blocks that allow precise rotations but they generally do not use the block model system and are actually tile entities or entity models (worse performance).
